tensorboard/plugins/scalar/BUILD

```diff
@@ -15,13 +15,10 @@ py_library(
     srcs = ["scalars_plugin.py"],
     srcs_version = "PY2AND3",
     deps = [
-        "//tensorboard:errors",
```

Comment:
`//tensorboard:errors` should stay, as `scalars_plugin.py` imports it.
(But the removals for the test targets are correct, because the test
targets don’t themselves import it.) The tests are passing because this
happens to get pulled in transitively, but for strict-deps purposes
(a.k.a., include-what-you-use) this should still be included.

Reply:
Restored

Reply:
Restored.